Resolve #21 posting Sample Post

Enoch-Kim opened the pull request:
* Post Sample
* 블로그 만드는 법 포스팅
Resolve #21

Enoch-Kim committed:
* Post Sample
* 블로그 만드는 법 포스팅

Enoch-Kim committed:
* Post Sample
* 블로그 만드는 법 포스팅

Enoch-Kim committed:
fix post
* add result
* add result capture image

Enoch-Kim committed:
fix post
* add result
* add result capture image

MOBUMIN reviewed (dismissed):
대표이미지, 태그, author 정보만 들어가면 완-벽! 일단 샘플로 그냥 업로드 해두고 수정해도 될 것 같아용!

jeongukjae commented on _posts/2020-09-28-make-blog-by-jekyll.md:
```diff
@@ -0,0 +1,163 @@
+image: assets/images/11.jpg
+description: "DSC 블로그를 만들며 정리한 내용."
+featured: true
+hidden: true
```
hidden 속성은 필요 없어보입니다.

jeongukjae commented on _posts/2020-09-28-make-blog-by-jekyll.md:
```diff
@@ -0,0 +1,163 @@
+layout: post
+title:  "Jekyll을 사용한 나만의 블로그 만들기"
+author: Enoch-Kim
+categories: [ Jekyll, tutorial ]
+tags: [red, yellow]
```
category, tags는 정리될 예정인가요??

Enoch-Kim replied:
아 이건 아직 태그 등등이 정리되지 않아 남겨뒀었는데,
미리 정리할까요??

jeongukjae replied:
나중에 정리될 예정이면 괜찮은 것 같아요!! ㅋㅋㅋ

Enoch-Kim replied:
이미 정리해서 푸시했습니다 😁
확인 부탁드려요~

jeongukjae commented on _posts/2020-09-28-make-blog-by-jekyll.md:
````diff
@@ -0,0 +1,163 @@
+(Mac의 경우 xcode가 설치돼있기 때문에 ruby도 함께 설치되어 있어 바로 사용할 수 있습니다.
+
+만약 설치가 안되어 있다면 terminal창을 열고 다음의 command를 통해 xcode development tool과 ruby를 설치해 주세요.)
+```
````
```suggestion
```sh
```

jeongukjae commented on _posts/2020-09-28-make-blog-by-jekyll.md:
```diff
@@ -0,0 +1,163 @@
+이제 repository를 보면 jekyll 폴더들이 올라와 있음을 볼 수 있습니다.
+
+***
+
+
+
```
이런 필요없는 라인은 지워주세요!! ㅋㅋㅋ

jeongukjae commented on _posts/2020-09-28-make-blog-by-jekyll.md:
````diff
@@ -0,0 +1,163 @@
+관심이 있으면 한번씩 구글링 해보셔서 사용하시는 것을 권장드립니다.**
+
+이제 jekyll server를 실행해봅시다.
+```
+jekyll serve
+#Configuration file: /Users/enoc/github.com/Enoch-Kim/enoch-kim.github.io/_config.yml
+#            Source: /Users/enoc/github.com/Enoch-Kim/enoch-kim.github.io
+#       Destination: /Users/enoc/github.com/Enoch-Kim/enoch-kim.github.io/_site
+# Incremental build: disabled. Enable with --incremental
+#      Generating... 
+#                    done in 0.549 seconds.
+# Auto-regeneration: enabled for '/Users/enoc/github.com/Enoch-Kim/enoch-kim.github.io'
+#    Server address: http://127.0.0.1:4000/mediumish-theme-jekyll/
+#  Server running... press ctrl-c to stop.
+
+```
````
```suggestion
```sh
$ jekyll serve
Configuration file: /Users/enoc/github.com/Enoch-Kim/enoch-kim.github.io/_config.yml
            Source: /Users/enoc/github.com/Enoch-Kim/enoch-kim.github.io
       Destination: /Users/enoc/github.com/Enoch-Kim/enoch-kim.github.io/_site
Incremental build: disabled. Enable with --incremental
      Generating... 
                    done in 0.549 seconds.
Auto-regeneration: enabled for '/Users/enoc/github.com/Enoch-Kim/enoch-kim.github.io'
    Server address: http://127.0.0.1:4000/mediumish-theme-jekyll/
   Server running... press ctrl-c to stop.
```
```

이런식으로 쓰는 것이 훨씬 좋을 것 같아요!

Enoch-Kim replied:
제가 md알못인지라 (머쓱..)

Enoch-Kim replied:
이런 기능이 있었군요!! 써보니 감탄..

Enoch-Kim committed:
Accept Review
* 2020-09-28 post
    change tag
    change image to jekyll logo
    change lines to <br/> tag
    code box to sh code box
    remove useless empty line
* post.html
    remove End Categories Comment
* add jekyll-logo.png

Enoch-Kim committed:
Accept Review
* 2020-09-28 post
    change tag
    change image to jekyll logo
    change lines to <br/> tag
    code box to sh code box
    remove useless empty line
* post.html
    remove End Categories Comment
* add jekyll-logo.png

jeongukjae commented on _layouts/post.html:
```diff
@@ -132,4 +131,4 @@ <h3 class="font-weight-bold">Summary</h3>
 {% endif %}
 <!--End Comments
-================================================== -->
+================================================== -->
```
```suggestion
================================================== -->

```

끝부분은 언제나 newline 넣어주세요!

jeongukjae commented on _posts/2020-09-28-make-blog-by-jekyll.md:
```diff
@@ -0,0 +1,151 @@
+featured: true
+---
+
+Mac 환경 기준입니다.<br/>
+(Mac의 경우 xcode가 설치돼있기 때문에 ruby도 함께 설치되어 있어 바로 사용할 수 있습니다.
+만약 설치가 안되어 있다면 terminal창을 열고 다음의 command를 통해 xcode development tool과 ruby를 설치해 주세요.)
```
```suggestion
MacOS 환경 기준입니다. MacOS의 경우 XCode가 설치돼있기 때문에 Ruby도 함께 설치되어 있어 바로 사용할 수 있습니다.
만약 설치가 안되어 있다면 터미널 창을 열고 다음의 명령어를 통해 XCode Development Tool과 Ruby를 설치해 주세요.
```

jeongukjae replied:
이렇게 쓰면 어떨까요??

jeongukjae commented on _posts/2020-09-28-make-blog-by-jekyll.md:
````diff
@@ -0,0 +1,151 @@
+Mac 환경 기준입니다.<br/>
+(Mac의 경우 xcode가 설치돼있기 때문에 ruby도 함께 설치되어 있어 바로 사용할 수 있습니다.
+만약 설치가 안되어 있다면 terminal창을 열고 다음의 command를 통해 xcode development tool과 ruby를 설치해 주세요.)
+```sh
````
```suggestion

```sh
```

이런건 항상 아래위로 빈칸을 남겨주세요!

jeongukjae commented:
master branch에서 rebase 했습니다!

jeongukjae commented on _posts/2020-09-28-make-blog-by-jekyll.md:
````diff
@@ -0,0 +1,151 @@
+(Mac의 경우 xcode가 설치돼있기 때문에 ruby도 함께 설치되어 있어 바로 사용할 수 있습니다.
+만약 설치가 안되어 있다면 terminal창을 열고 다음의 command를 통해 xcode development tool과 ruby를 설치해 주세요.)
+```sh
+# install xcode development tool
+xcode-select --install
+
+# if home brew does not exist
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+# end if
+
+# install ruby & setting env variable
+brew install ruby
+echo 'export PATH="/usr/local/opt/ruby/bin:$PATH"' >> ~/.bash_profile
+
+# check ruby
+which ruby
+# /usr/local/opt/ruby/bin/ruby 가 나와야 합니다.
+ruby -v
````
```suggestion
# Xcode Development Tool이 없다면 아래처럼 설치해주세요.
xcode-select --install

# homebrew가 없다면 아래처럼 설치해주세요.
/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"

# Ruby가 없다면 아래처럼 설치해주세요.
brew install ruby
echo 'export PATH="/usr/local/opt/ruby/bin:$PATH"' >> ~/.bash_profile

# check ruby
which ruby
# /usr/local/opt/ruby/bin/ruby 가 나와야 합니다.
ruby -v
```


위처럼 수정하는 것은 어떨까요?? 한국어 독자이니 이렇게 쓰는 것이 좋을 것 같아요.

jeongukjae commented on _posts/2020-09-28-make-blog-by-jekyll.md:
````diff
@@ -0,0 +1,151 @@
+# /usr/local/opt/ruby/bin/ruby 가 나와야 합니다.
+ruby -v
+```
+우선 ruby를 통해 jekyll을 설치해줍니다. 
````
```suggestion

우선 ruby를 통해 jekyll을 설치해줍니다. 

```

jeongukjae reviewed (changes requested):
지금은 이정도가 있는데요, 아마 마크다운 스타일을 한번 전체적으로 맞추어야 할 것 같아요. 그 부분만 제가 커밋을 하나 넣겠습니다.

Enoch-Kim committed:
Merge branch 'issue21/post-sample' of github.com:DSC-University-of-Seoul/dsc-university-of-seoul.github.io into issue21/post-sample

Enoch-Kim committed:
MarkDown 형식 정리
* 2020-09-28-make-blog-by-jekyll.md
    code box 앞뒤 new line추가
    문장 끝 빈공간 제거
    파일 마직막 new line 1개로
    단어 정식명칭 적용
    키원드에 back quote 적용

MOBUMIN commented on _posts/2020-09-28-make-blog-by-jekyll.md:
```diff
@@ -0,0 +1,172 @@
+featured: true
+---
+
+macOS 환경인 기준입니다.
```
```suggestion
macOS 환경 기준입니다.
```
환경인 이라는 말 따로 없는거 맞겠쥬 ..? 헤헤!

MOBUMIN commented on _posts/2020-09-28-make-blog-by-jekyll.md:
````diff
@@ -0,0 +1,172 @@
+`block in materialize': Could not find public_suffix-3.0.3 in any of the sources (Bundler::GemNotFound)
+```
+
+당황하지 말고 `index.html` 혹은 `index.md`를 열고 다음의 사진에서 `jekyll-seo-tag` 와 `jekyll-archives` 그리고 `jekyll-feed`를 제거해주세요.
````
```suggestion
당황하지 말고 `index.html` 혹은 `index.md`를 열고 다음의 사진에서 `jekyll-seo-tag` 와 `jekyll-archives`를 제거해주세요.
```
jekyll-feed는 없애지 않는 것으로...!!!

Enoch-Kim replied:
꼼꼼한 리뷰 감사합니다~~

MOBUMIN commented on _posts/2020-09-28-make-blog-by-jekyll.md:
```diff
@@ -0,0 +1,172 @@
+
+![Capture](../assets/images/capture-jekyll-success.png)
+
+이제 한번 자신의 웹사이트에 접속해 봅시다. `{username}.github.io` 로 접속해보시면 됩니다. (저의 경우 `https://enoch-kim.github.io`)
```
```suggestion
이제 한번 자신의 웹사이트에 접속해 봅시다. `{username}.github.io` 로 접속해보시면 됩니다. (저의 경우 [https://enoch-kim.github.io](https://enoch-kim.github.io))
```
실제로 유현님의 주소를 링크해주시면 어떨까.. 하는 작은 의견 조심스레 적어봅니다..ㅎㅎ

Enoch-Kim replied:
그건 다음 포스트때 해야해요 저 이 다음 진행 안해서 이미지랑 깨지거든요...ㅋㅋㅋㅋ

Enoch-Kim replied:
TODO로 달아놓겠습니다.

Enoch-Kim committed:
Accept Review
** post 수정
    - 오타 제거
    - jekyll-feed 관련 내용 수정
    - TODO 추가

MOBUMIN reviewed (dismissed):
LGTM :D 👍

Enoch-Kim committed:
blog post 내용 추가
* resolve TODOs
* baseurl 변경 내용 추가
* google analytics 설정 내용 추가
* disqus 설정 내용 추가

MOBUMIN reviewed (changes requested):
analytics 사진에 코드 부분에 gtag로 아이디가 보입니다..!!

+ disqus 한국어 부분은 따로 편법을 써서 구현할 수 있더라구요!(개발자 도구로 'ko'추가해서 설정만 하면 끗..) 이 내용은 본문에 안 들어가도 괜찮으려나요?

Enoch-Kim commented:
> analytics 사진에 코드 부분에 gtag로 아이디가 보입니다..!!
> 
> * disqus 한국어 부분은 따로 편법을 써서 구현할 수 있더라구요!(개발자 도구로 'ko'추가해서 설정만 하면 끗..) 이 내용은 본문에 안 들어가도 괜찮으려나요?

강제 푸쉬해야겠네요..
ko 부분은 혹시 관련 글 포워딩해주실 수 있을까요?

Enoch-Kim committed:
이미지 추가
* disqus image
* google analytics image

Enoch-Kim committed:
Language부분 내용 변경
* 처음에 야매로 한국어 설정하려했으나
  React로 바뀌면서 막힘

Enoch-Kim committed:
Disqus 설명 괄호 추가

MOBUMIN reviewed (dismissed):
LGTM

jeongukjae committed:
revert _layouts/post.html
